Resources/Prototypes/Entities/Structures/Power/Generation/Singularity/singularity.yml

```diff
@@ -13,7 +13,17 @@
     bodyType: Dynamic
   - type: Fixtures
     fixtures:
-    - id: EventHorizon
+    - id: EventHorizonCollider
+      shape:
+        !type:PhysShapeCircle
+          radius: 0.5
+      restitution: 0.9
+      density: 99999
+      mask:
+      - AllMask
+      layer:
+      - AllMask
+    - id: EventHorizonConsumer
       shape:
         !type:PhysShapeCircle
           radius: 0.5
```

Comment:
Yeah. One of them is to take care of bouncing off of the containment field. The other is to take care of eating things. They can't be the same fixture because if it's not hard the singularity will just phase through the containment field and if it's hard the singularity will bounce off of things it eats. AFAIK there is no way to make a fixture selectively hard with the current system.

Reply:
What do you want me to do about the fixtures? If I just use a single fixture it has to be hard because otherwise it would just phase through the containment field. If it's hard it will bounce off of walls and the like, which is what this PR is trying to address.